src/test/java/com/theelixrlabs/healthcare/HealthCarePocApplicationTests.java

```diff
@@ -6,7 +6,7 @@
 @SpringBootTest
 class HealthCarePocApplicationTests {
 
-	@Test
+	//@Test
```

Comment:
Why the default test class annotation is commented? May I know the reason? Avoid committing commented codes.

Reply:
Uncommented the test annotation and commited the code.